build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -50,17 +48,18 @@
       MuseScore can import and export MIDI and MusicXML files, and it can also import from Capella and several other programs. MuseScore can export to PDF, PNG, and other graphic formats, to WAV and other audio formats, or to Lilypond for an alternative layout and print option.
     </p>
   </description>
-
+  <kudos>
```

Comment:
① What are these for? Why do you give those things kudos?

② This tag is, according to the appstreamcli validator, a GNOME-specific extension and could be removed.

Reply:
[①](https://github.com/flathub/flathub/wiki/AppData-Guidelines#kudos)

② Yes, but with it MuseScore gets a nice ‘Documentation’ badge in GS, there’s really no reason to remove it:

![obrazok](https://user-images.githubusercontent.com/7472425/49338696-5f6cf880-f625-11e8-99c4-cb623eb85616.png)

Also, for example in Flatpak the kudos are added automatically into `org.musescore.MuseScore.xml.gz`, without it it produces an incomplete list.

Reply:
Can you patch them into the XML file during the Flatpak build, then, and leave them out for the regular distro packaging?

That is, patch the file in some script under `build/` that creates the bundle.

Reply:
Surely it would be easier overall to keep these in and then patch them *out* if you don't want them?

Patching could be done by CMake during `configure_file()`. This could be used to patch them in, or to patch them out by enclosing them in comments tags (`<!--` and `-->`), or simply left to Debian's patch system.

Reply:
I’ve contacted the AppStream maintainers, they said the `appstreamcli` warning can be safely ignored.